neo/io/neomatlabio.py

```diff
@@ -343,6 +402,11 @@ def create_ob_from_struct(self, struct, classname):
                     data_complement["t_start"] = arr.min()
                 else:
                     data_complement["t_start"] = 0.0
+            if "spatial_scale" in (at[0] for at in cl._necessary_attrs):
+                if len(arr) > 0:
+                    data_complement["spatial_scale"] = arr
+                else:
+                    pass  #
```

Comment:
I would still prefer to come up with a better system for this. If we write an image sequence without giving this it would error. I don't use ImageSequence at all so I don't know what the appropriate default would be to put it (maybe just 1?), but all the others (like t_start) get a default in case they aren't found.

Reply:
I've put in a default of 1.0